pkg/network/ebpf/c/runtime/http.c

```diff
@@ -122,8 +122,8 @@ int kprobe__tcp_sendmsg(struct pt_regs *ctx) {
     return 0;
 }
 
-SEC("kretprobe/tcp_sendmsg")
```

Comment:
Is there a reason this remains a `kretprobe`? I think a `kprobe` might be sufficient now.

Reply:
It make sense as we want the notification happen after parsing the response